build.gradle

```diff
@@ -45,6 +46,12 @@ dependencies {
     implementation group: 'io.micrometer', name: 'micrometer-registry-prometheus', version: '1.8.2'
     implementation group: 'net.thisptr', name: 'jackson-jq', version: '1.0.0-preview.20210928'
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.12.5'
+    implementation 'io.jaegertracing:jaeger-client:1.8.0'
+    implementation 'io.opentelemetry:opentelemetry-sdk:1.11.0'
```

Comment:
May you consider using opentelemetry-bom the same way we use jackson-bom? We should promote this style.

Reply:
updated